### PR TITLE
Refactor resource console language and documentation

### PR DESCRIPTION
**Pull request by Diptipowervs**
### Community Note

* Please vote on this pull request by adding a 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to the original pull request comment to help the community and maintainers prioritize this request
* Please do not leave "+1" or other comments that do not add relevant new information or questions, they generate extra noise for pull request followers and do not help prioritize the request

Relates OR Closes #0000

Output from acceptance testing:

```
$ make testacc TESTARGS='-run=TestAccXXX'

...
```

### CODE REVIEW

File: ibm/service/power/resource_ibm_pi_instance_console_language.go [hunk 1]
```diff
@@ -32,12 +30,12 @@ func ResourceIBMPIInstanceConsoleLanguage() *schema.Resource {
...
+			Arg_CloudInstanceID: {
 				Type:        schema.TypeString,
 				Required:    true,
 				Description: "PI cloud instance ID",
 			},
-			helpers.PIInstanceName: {
+			Arg_InstanceName: {
 				Type:        schema.TypeString,
 				Required:    true,
 				Description: "The unique identifier or name of the instance",
```
Comment (ismirlia) [inline ibm/service/power/resource_ibm_pi_instance_console_language.go:41]: Create a new constant for PIConsoleLanguageCode and put into the constant file.

File: ibm/service/power/resource_ibm_pi_instance_console_language.go [hunk 2]
```diff
@@ -32,12 +30,12 @@ func ResourceIBMPIInstanceConsoleLanguage() *schema.Resource {
 
 		Schema: map[string]*schema.Schema{
 			// Required Attributes
-			helpers.PICloudInstanceId: {
+			Arg_CloudInstanceID: {
 				Type:        schema.TypeString,
```
Comment (ismirlia) [inline ibm/service/power/resource_ibm_pi_instance_console_language.go:34]: Sort all stanzas alphabetically and match their descriptions to what is on the documentation.

File: website/docs/r/pi_console_language.html.markdown [hunk 3]
```diff
@@ -22,14 +22,14 @@ resource "ibm_pi_console_language" "example" {
 }
 ```
 
-**Note**
```
Comment (ismirlia) [inline website/docs/r/pi_console_language.html.markdown:25]: Remove extra line of whitespace on line 2.

**Comment by ismirlia** [comment]
Whenever you get a chance @Diptipowervs look through my comments and apply these sort of changes to other PRs.

File: ibm/service/power/resource_ibm_pi_instance_console_language.go (hunk 1 above)
Comment (Diptipowervs) [reply ibm/service/power/resource_ibm_pi_instance_console_language.go:41]: Added

**Comment by Diptipowervs** [comment]
--- PASS: TestAccIBMPIInstanceConsoleLanguage (43.93s)
PASS

**Review by michaelkad — APPROVED** [review]
lgtm

**Comment by ismirlia** [comment]
Opened this: https://github.com/powervs-ibm/terraform-provider-ibm/pull/117 closing

**Pull request closed without merging**